mongo/content.md

```diff
@@ -37,7 +37,7 @@ See the [official docs](http://docs.mongodb.org/manual/) for infomation on using
 Just add the `--storageEngine` argument if you want to use the WiredTiger storage engine in MongoDB 3.0 and above without making a config file. Be sure to check the [docs](http://docs.mongodb.org/manual/release-notes/3.0-upgrade/#change-storage-engine-to-wiredtiger) on how to upgrade from older versions.
 
 ```console
-$ docker run --name some-mongo -d mongo --storageEngine=wiredTiger
+$ docker run --name some-mongo -d mongo mongod --storageEngine wiredTiger
```

Comment:
Gotcha. Well originally I was just fixing the `=` to be a space, and then it appeared that said change didn't work without also adding the binary. Seems like it does indeed work, so I'll remove the binary. My bad.